src/main/drivers/compass/compass_lis3mdl.c

```diff
@@ -101,6 +107,53 @@
 #define LIS3MDL_FAST_READ           0x80  // Default 0
 #define LIS3MDL_BDU                 0x40  // Default 0
 
+#ifdef USE_MAG_DATA_READY_SIGNAL
```

Comment:
Do you have plans to add support for MAG IRQ processing? 
Currently this functionality is unused (for  all drivers), the code will only slowly rot ...

Reply:
We found that out the hard way. 

We do have the drdy pin in use by the sensor on our FC, but as you mentioned. This seems to not be supported yet. Our config.h is available here if you have any insight or suggestions. 

https://github.com/HeronPrecision/config/blob/heron_fc1rc2/configs/HERONPRECISION_FC1RC2/config.h